content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md

```diff
@@ -10,4 +10,4 @@ stages:
     defaultValue: false
     fromVersion: "1.27"
 ---
-Enables in-place Pod vertical scaling.
```

Comment:
I like the existing text. You can't vertically scale on other than Pod resource limits - in that it wouldn't make sense to.

Reply:
How would you feel about "in-place vertical scaling of Pod resource limits"?